src-plugins/manualRegistration/manualRegistrationToolBox.cpp

```diff
@@ -334,6 +361,8 @@ void manualRegistrationToolBox::reset()
     viewFuse->removeLayer(1);
     viewFuse->addLayer(viewMoving->layerData(0));
     synchroniseMovingFuseView();
+    d->b_save->setDisabled(true);
+    d->b_saveTransformation->setDisabled(true);
```

Comment:
You can do a boolean function for that because it's done here, in computeRegistration and in the constructor :)